clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -0,0 +1,53 @@
+import {
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+  Input,
+  Select,
+  SimpleGrid,
+} from "@fidesui/react";
+import { FieldHookConfig, useField } from "formik";
+
+interface InputProps {
+  label: string;
+}
+
+export const CustomTextInput = ({
+  label,
+  ...props
+}: InputProps & FieldHookConfig<string>) => {
+  const [field, meta] = useField(props);
+  const { type, placeholder } = props;
+  const isInvalid = !!(meta.touched && meta.error);
+  return (
+    <FormControl isInvalid={isInvalid}>
+      <SimpleGrid columns={[1, 2]}>
+        <FormLabel htmlFor={props.id || props.name} size="sm">
+          {label}
+        </FormLabel>
+        <Input {...field} type={type} placeholder={placeholder} size="sm" />
+      </SimpleGrid>
+      {isInvalid && <FormErrorMessage>{meta.error}</FormErrorMessage>}
+    </FormControl>
+  );
+};
+
+export const CustomSelect = ({
+  label,
+  ...props
+}: InputProps & FieldHookConfig<string>) => {
+  const [field, meta] = useField(props);
+  const isInvalid = !!(meta.touched && meta.error);
+  return (
+    <FormControl isInvalid={isInvalid}>
+      <SimpleGrid columns={[1, 2]}>
+        <FormLabel htmlFor={props.id || props.name} size="sm">
+          {label}
+        </FormLabel>
+        {/* @ts-ignore having trouble getting Formik and Chakra select to be happy together */}
+        <Select {...field} {...props} size="sm" />
+      </SimpleGrid>
+      {isInvalid && <FormErrorMessage>{meta.error}</FormErrorMessage>}
```

Comment:
^ Not that I'm always against the short-circuiting, but maybe just something to be aware of?

Reply:
yeah, I'm glad you brought it up! I'll keep it in mind for the future. I'm happy to change these (though I believe in these cases they are safe since they are booleans so won't render random `0`s)

Reply:
Thanks for looking! I'm fine approving if you're confident it won't run into the errors mentioned in that post. 😄